src/app/app.component.ts

```diff
@@ -0,0 +1,29 @@
+import { TranslateService } from '@ngx-translate/core';
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { LoaderService } from './core/services/loader.service';
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+})
+export class AppComponent {
+  title = 'pms-application';
```

Comment:
It better to use modifiers and typing for each property or method. It leads to better readability of code.

```public title: string = 'pms-application'```